addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-ais.yaml

```diff
@@ -0,0 +1,17 @@
+#@ load("/values.star", "values")
+#@ load("@ytt:overlay", "overlay")
+
+#@overlay/match by=overlay.subset({"kind": "AviInfraSetting"})
```

Comment:
You can check if the network name for the control plane is the same as the data network then decide to apply a AIS or not.
You can do something like `ingressclass`here: 
https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-statefulset.yaml#L288

https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/overlays/overlay-statefulset.yaml#L305